### PR TITLE
Remove Books from New Bookshelf Workflow

### LINKED ISSUE #327: Create a workflow that will remove books form the New Bookshelf by a list of barcodes
This workflow needs to take a list of barcodes and for each one it should check to see if that item has been at the Evans nbs location for more than 30 days. If it has it should set both the temporary location and the temporary loan type to null.

Input:
1. User provided CSV file containing list of all barcodes to be checked.

Add an email - resulting list is going to be a subset of the original csv.

All those books that will be removed from the nbs

`The attachment contains the list that has been removed from the new bookshelf`

File Upload - CSV file containing list of barcodes
 - include in query - to check each bar

### PR DESCRIPTION
**Pull request by rmathew1011**
Resolves #327 

Please review https://github.com/TAMULib/fw-registry/pull/342

### CODE REVIEW

**Review by qtamu — DISMISSED** [review]

**Comment by Dbreck-TAMU** [comment]
Unable to approve as I've directly contributed to this code in the past.

File: README.md [hunk 1]
```diff
@@ -457,3 +457,42 @@ curl --location --request POST 'http://localhost:9001/mod-workflow/events/workfl
...
+```shell
+fw config set okapi-internal ***
+fw config set bcn-mail-from ***
+```
+
+These variables are required when triggering the workflow:
+
+| Variable Name  | Allowed Values | Short Description |
+| -------------- | -------------- | ----------------- |
+| path           | directory path | The directory on the system where the CSV file is stored within on the server and contain the `tenantPath` (include trailing slash after the directory). |
```
Comment (Dbreck-TAMU) [inline README.md:474]: Path is used here, but the workflow uses an input called `inputFilePath`. 

Please double check that this lines up w/ Catalog Reports, which uses `path` and that the documentation/workflow/UI params match.

Comment (wwelling) [reply README.md:474]: `inputFilePath` is a variable name added by mod-workflow event endpoint for file upload. The path variable here is provided during the event POST to inform mod-workflow where to store the file relatively. It is something we should actually avoid as mod-workflow and a event request should know the tenant and workflow to build the full path to store the upload file.

**Comment by jmicah** [comment]
This is going to be addressed in a different branch.

**Pull request closed without merging**